### PR TITLE
MISP-1266: Remove incorrect license identifiers

### PR DESCRIPTION
**Pull request by shanx**

### CODE REVIEW

**Comment by robinmdh** [comment]
this is open source and here the license is valid IMHO

**Comment by robinmdh** [comment]
I'll make a suggestion for that change.

File: build_for_ultimaker.sh [hunk 1]
```diff
@@ -1,7 +1,5 @@
 #!/bin/sh
 #
-# SPDX-License-Identifier: AGPL-3.0+
-#
```
Comment (robinmdh) [inline build_for_ultimaker.sh:4]:
```suggestion
# SPDX-License-Identifier: LGPL-3.0+ 
#
```

Comment (CoenSchalkwijk) [reply build_for_ultimaker.sh:4]: +1 
Not only that: this file is used in multiple repositories and already marked as LGPL.

**Review by CoenSchalkwijk — CHANGES_REQUESTED** [review]

**Review by bryanfoley — APPROVED** [review]

**Review by CoenSchalkwijk — APPROVED** [review]

**Pull request merged**